scripts/xonsh-cat

```diff
@@ -1,3 +1,9 @@
-#!/usr/bin/env python3 -u
+#!/bin/sh
+'''#'
+exec python3 -u "$0" "$@"
+'''#'
+# Portable trick to pass option [-u] to python3, more portable than /usr/bin/env.
+# vim: syn=python
```

Comment:
I don't think we need the vim syntax hint in here

Reply:
removed.